Dockerfile

```diff
@@ -92,19 +91,26 @@ RUN mkdir -p src/backend/src \
     && mkdir -p src/cycles_ledger/types/src \
     && touch src/cycles_ledger/types/src/lib.rs \
     && mkdir -p src/shared/src \
```

Comment:
The command below is essentially the same, just updated.  The goal: We build, so that the cache is warm.